packages/ui/src/components/shared/menu/ImportAccountDialog/TransferAccountBackup.tsx

```diff
@@ -0,0 +1,123 @@
+import { useToggle } from 'react-use';
+import { AccountBackup } from '@coong/keyring/types';
+import { Alert, Button, TextField } from '@mui/material';
+import AccountCard from 'components/pages/Accounts/AccountCard';
+import useAccountBackupValidation from 'components/shared/menu/ImportAccountDialog/useAccountBackupValidation';
+import EmptySpace from 'components/shared/misc/EmptySpace';
+import useHighlightNewAccount from 'hooks/accounts/useHighlightNewAccount';
+import useAccountNameValidation from 'hooks/useAccountNameValidation';
+import { useWalletState } from 'providers/WalletStateProvider';
+import { Props } from 'types';
+
+interface TransferAccountBackupProps extends Props {
+  backup: AccountBackup;
+  onClose: () => void;
+  resetBackup: () => void;
+}
+
+function TransferAccountBackup({ backup, resetBackup, onClose }: TransferAccountBackupProps): JSX.Element {
+  const { keyring } = useWalletState();
+  const { t } = useTranslation();
+  const [onImporting, setOnImporting] = useToggle(false);
+  const [newName, setNewName] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const { validation } = useAccountNameValidation(newName);
+  const { setNewAccount } = useHighlightNewAccount();
+  const { conflict, resolvable, accountInfo } = useAccountBackupValidation(backup);
+
+  const originalAccountBackupName = backup.meta.name as string;
+
+  if (conflict && !resolvable)
+    return (
+      <div>
+        <AccountCard account={accountInfo} />
+        <Alert severity='error' className='mt-4'>
+          {t<string>(conflict, { name: originalAccountBackupName })}
+        </Alert>
+        <Button onClick={resetBackup} className='mt-4' fullWidth>
+          {t<string>('Back')}
+        </Button>
+      </div>
+    );
+
+  const doImportAccount = (event: FormEvent) => {
+    event.preventDefault();
+
+    setOnImporting(true);
+
+    setTimeout(async () => {
+      try {
+        // To avoid change the content of original backup
+        const cloneBackup: AccountBackup = { ...backup, meta: { ...backup.meta } };
+        cloneBackup.meta.name = newName || originalAccountBackupName;
+
+        const account = await keyring.importAccount(cloneBackup, password);
+        onClose();
+        toast.success(t<string>('Import account successfully'));
+        setNewAccount(account);
+      } catch (e: any) {
+        setOnImporting(false);
+        toast.error(t<string>(e.message));
+      }
+    }, 200);
+  };
+
+  const handleNewNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const newName = e.currentTarget.value;
+    setNewName(newName);
+
+    // To display the preview with the new name
+    // without altering the original backup name
+    if (!newName) accountInfo.name = originalAccountBackupName;
+    else accountInfo.name = newName;
+  };
+
+  return (
+    <div>
+      <AccountCard account={accountInfo} />
+      {conflict && (
+        <Alert severity='info' className='mt-4'>
```

Comment:
We should use `warning` here instead of `info` to clarify that the conflict must be addressed before continuing + adjust spacing a bit.

```suggestion
        <Alert severity='warning' className='mt-6'>
```